helpers/press-release-sparql-queries.js

```diff
@@ -138,3 +154,52 @@ export async function createPublicationTasksPerPublicationChannel(graph, pressRe
 		throw err;
 	}
 }
+
+export function getPublicationChannelsByPublicationEvent(graph, publicationEvent) {
+	return query(`
+		${PREFIXES}
+		
+		SELECT ?pubChannel
+		WHERE {
+			GRAPH ${sparqlEscapeUri(graph)} {
+				${sparqlEscapeUri(publicationEvent)}	ext:publicationChannels 	?pubChannel .
```

Comment:
Dit is niet het correcte predicaat om de publication channels die bij een publication-event horen te selecteren. Zie ook https://github.com/kanselarij-vlaanderen/kaleidos-documentation/blob/master/data-model/press-releases.md